### DISCUSSION TITLE
Empty page when querying the database

### DISCUSSION THREAD

**Discussion by ilyaZ45** [sidorares/node-mysql2#1509] [General]
Hello.Tell me?I can't deal with DB requests.There is output to all columns of the table during the query.If I refer to a specific table header, then an empty result is obtained.
![image](https://user-images.githubusercontent.com/51793326/153420776-26b5e569-3a2d-4620-83cf-b3e6f0fd8193.png)

If I refer to the entire table, it is displayed normally
![image](https://user-images.githubusercontent.com/51793326/153420823-e6ebeeac-a3d0-463e-b5b5-049c9ea99144.png)

**Comment by sidorares** [comment]
what do you mean by empty? An empty array in `results` variable? What is the output for `"SELECT * FROM table_vue"` query?

**Comment by ilyaZ45** [comment]
if `SELECT * FROM table_vue`
![image](https://user-images.githubusercontent.com/51793326/153531762-3aee4fd0-25b6-4e7c-bd2e-eec0e64523a1.png)
What change if  input `id`?Why i get empty?
 `SELECT id FROM table_vue
        if(err) {
                    console.log(err);
                    result(err, null);
                } else {
                    result(null,results['id']);
                }
            });`
![image](https://user-images.githubusercontent.com/51793326/153533530-c283d743-2cd5-4adb-8f54-5cdeb54093d5.png)

**Comment by sidorares** [comment]
results is an array of objects with `id` property on them, similar to `[{id: '1'}, {id: '2'}, {id: '3'}]` so I'd check first `results['id']` - this seems incorrect to me
